QueueManagerTest/DirectoryReaderTests.cs

```diff
@@ -1,26 +1,44 @@
 ﻿using System.Collections.Generic;
 using NUnit.Framework;
 using QueueManager;
+using System.IO;
+using System;
 
 namespace QueueManagerTest
 {
     [TestFixture]
-    public class NetflixReaderTests
+    public class DirectoryReaderTests
     {
         private DirectoryReader _directoryReader;
-        private string path;
+        private string upperLevelFolderName;
+        private string pathString;
+        private string appendedPathString;
+        private List<string> filesInUpperLevelFolder;
 
         [SetUp]
         public void SetUp()
         {
             _directoryReader = new DirectoryReader();
-            path = @"..\..\TestDir\";
+            upperLevelFolderName = @"..\..\TestDir\";
+            pathString = Path.GetFullPath(upperLevelFolderName);
+            Directory.CreateDirectory(pathString);
+            filesInUpperLevelFolder = new List<string> { "a1.iso", "a2.iso", "a3.iso", "a4.iso - Shortcut.lnk" };
+            foreach (string file in filesInUpperLevelFolder)
+            {
+                appendedPathString = Path.Combine(pathString, file);
+                File.Create(appendedPathString);
+            }
         }
 
         [Test]
         public void TestAddFilesInDirectoryToList()
```

Comment:
if you want something a little shorter you could do something like this:

`var testList = filesInUpperLevelFolder.Select(x => $"{upperLevelFolderName}{x}");`

instead of the foreach loop